src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -0,0 +1,246 @@
+          nfs_items,
+          btrfs_items,
+          tmpfs_items,
+          unused_items
+        ]
+      end
+
+      def harddisk_items
+        Item(
+          Id(:hd),
+          term(:icon, Icons::HD),
+          _("Hard Disks"),
+          open?(:hd),
+          disks_items
+        )
+      end
+
+      def disks_items
+        @device_graph.disks.map do |disk|
+          Item(
+            Id(disk.name),
+            disk.sysfs_name,
+            open?(disk.name),
+            partition_items(disk)
+          )
+        end
+      end
+
+      def partition_items(disk)
+        disk.partitions.map do |partition|
+          Item(
+            Id(partition.name),
+            partition.sysfs_name,
+            open?(partition.name)
+          )
+        end
+      end
+
+      def raid_items
+        Item(
+          Id(:raid),
+          term(:icon, Icons::RAID),
+          _("RAID"),
+          open?(:raid),
+          [] # TODO: real MD subtree
+        )
+      end
+
+      def lvm_items
+        Item(
+          Id(:lvm),
+          term(:icon, Icons::LVM),
+          _("Volume Management"),
+          open?(:lvm),
+          lvm_vgs_items
+        )
+      end
+
+      def lvm_vgs_items
+        @device_graph.lvm_vgs.map do |vg|
+          Item(
+            Id(vg.vg_name),
+            vg.vg_name,
+            open?(vg.vg_name),
+            lvm_lvs_items(vg)
+          )
+        end
+      end
+
+      def lvm_lvs_items(vg)
+        vg.lvm_lvs.map do |lv|
+          Item(
+            Id(lv.name),
+            lv.lv_name,
+            open?(lv.name)
+          )
+        end
+      end
+
+      def crypt_files_items
+        Item(
+          Id(:loop),
+          term(:icon, Icons::LOOP),
+          _("Crypt Files"),
+          open?(:loop),
+          [] # TODO: real subtree
+        )
+      end
+
+      def device_mapper_items
+        Item(
+          Id(:dm),
+          term(:icon, Icons::DM),
+          _("Device Mapper"),
+          open?(:dm),
+          [] # TODO: real subtree
+        )
+      end
+
+      def nfs_items
+        Item(
+          Id(:nfs),
+          term(:icon, Icons::NFS),
+          _("NFS"),
+          open?(:nfs),
+          [] # TODO: real subtree
+        )
+      end
+
+      def btrfs_items
+        Item(
+          Id(:btrfs),
+          term(:icon, Icons::NFS),
+          _("Btrfs"),
+          open?(:btrfs),
+          [] # TODO: real subtree
+        )
+      end
+
+      def tmpfs_items
+        Item(
+          Id(:tmpfs),
+          term(:icon, Icons::NFS),
+          _("tmpfs"),
+          open?(:tmpfs),
+          [] # TODO: real subtree
+        )
+      end
+
+      def unused_items
+        Item(
+          Id(:unused),
+          term(:icon, Icons::UNUSED),
+          _("Unused Devices"),
+          open?(:unused),
+          [] # TODO: real subtree
+        )
+      end
+
+      def open?(id)
+        @opened.include?(id)
+      end
+
+      def term(*args)
```

Comment:
in fact it is in Yast.term as common place, but I am testing it on old system, so it do not work. Or add it also to UIShortcuts